### DISCUSSION TITLE
Group dependencies updates in a single PR while also keeping individual PRs

### DISCUSSION THREAD

**Discussion by Marsup** [renovatebot/renovate#22148] [Suggest an Idea]
**What would you like Renovate to be able to do?**

Mirroring the request in https://github.com/renovatebot/config-help/issues/95, I would like renovate to both open individual PRs and grouped PRs, to ease the bump of many dependencies at once yet still allowing (and testing) individual bumps.

**Describe the solution you'd like**

I think a nice and flexible solution to this would be a setting for the groups, like `"metaGroup": true` or `"capture": false` (name unimportant), that would create a group and do PRs with it, but not interrupt the flow and still consider the other existing groups. This would allow for example to have an "eslint" group and also have a group "dev dependencies" with an overlap between the two.

**Describe alternatives you've considered**

I guess we could also imagine checkboxes in the master issue so that renovate can create a tailored PR with just the bumps you want, but that seems complicated isn't it ?

**Additional context**

I'd consider this feature to be part of the noise reduction even though it could be seen as more noise, but reducing the number of actions needed to merge a lot of PRs seem worth it.

**Comment by rarkins** [comment]
An idea I’ve been thinking about for a while is to “roll up” passing updates into a single PR. The idea is similar to yours but I had thought it would be least noisy to first test in a branch only (no PR) and then add to the rollup/meta PR once passing. 

What do you think?

**Comment by Marsup** [comment]
It seems like a good idea, although I do like to have control over the meta grouping. Just because a PR is green for a dependency doesn't mean I want to merge it with the others.

To explain further, I guess what I'm trying to achieve is having the right balance between mass merging and selective merging because I can't find a grouping that makes sense, since it very much depends on the updates contained in large groups like dev deps. Sometimes I want all dev deps because it contains all and only what I want, sometimes there's an extra dep in there that I don't want so I would then merge individual PRs.

The alternative I suggested in the OP seems perfect but a massive headache for you.

**Comment by geraintwhite** [comment]
The ability to “roll up” passing updates into a single PR would be a fantastic noise-reducing feature!

**Comment by HonkingGoose** [comment]
@rarkins This issue seems similar/related to #4404. Do you want to keep both issues around?
